web/src/stream_muting.js

```diff
@@ -10,7 +10,9 @@ export function update_is_muted(sub, value) {
     // TODO: In theory, other message lists whose behavior depends on
     // stream muting might need to be live-updated as well, but the
     // current _all_items design doesn't have a way to support that.
-    message_lists.home.update_muting_and_rerender();
+    if (message_lists.current?.data.filter.is_in_home()) {
+        message_lists.current.update_muting_and_rerender();
+    }
```

Comment:
I think this one might need to have a loop over `all_rendered_message_lists` checking their filters for `is_in_home` -- since we're not currently committed to remove the feature that we can have more than one rendered in the DOM at a time, we can't assume that `home` is current.

Reply:
Updated!